### ISSUE TITLE
@AllowConcurrentEvents should be used

### ISSUE THREAD

**Issue by yososs** [bennidi/eventbus-performance#2]
When parallel performance comparison, should you use @AllowConcurrentEvents annotation for Guava?

**Comment by bennidi** [comment]
`@AllowConcurrentEvents` configures an event handler as eligible for multi-threaded invocation. This is the same as using `@Handler(dispatch = Invoke.asynchronously)` with MBassador. It's a different test scenario which is currently not implemented.

**Comment by yososs** [comment]
> @Handler(dispatch = Invoke.asynchronously)

Is not it the same as using Guava's AsyncEventBus?

> AsyncEventBus -> Async Dispatch
> EventBus           -> Sync Dispatch
> 
> @Subscribe @AllowConcurrentEvents -> Allow Concurrent Subscribe method
> @Subscribe                                           -> Disallow Concurrent Subscribe method(Synchronize using synchronized block)

**Comment by bennidi** [comment]
No. You have to draw a distinction between asynchronous publication (fire and forget) when publishing an event. And the invocation of handlers for a specific event publication (which can be synchronous or asynchronous).
